### PR TITLE
Add connector updates to the 2201.9.0 release note

### PR DESCRIPTION
**Pull request by Nuvindu**
## Purpose
Update the release note 2201.9.0 with the following connector updates.

> Google Calendar
> Docusign eSignature
> Docusign Admin
> Docusign Click
> Confluent Schema Registry
> Confluent Avro Serialization/Deserialization

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -262,22 +262,53 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
...
+
+#### `github` package
 
 - Introduced support for connecting to GitHub REST API version 2022-11-28, replacing the GitHub GraphQL API based connector.
 
-#### `guidewire.insnow` package 
+#### `googleapis.gcalendar` package
+
+- Enhanced the connector APIs by incorporating resource functions.
+- With the new changes the package name is changed to `googleapis.gcalendar` which was previously maintained under the name `googleapis.calendar`.
```
Comment (nipunayf) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:308]:
```suggestion
The package name has been changed to `googleapis.gcalendar` from its previous name, `googleapis.calendar`.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -262,22 +262,53 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
...
+
+#### `confluent.cregistry` package
+
+- Introduced support for integrating with the Confluent Schema Registry.
+
+#### `docusign.dsadmin` package
+
+- Enhanced the connector APIs to align with DocuSign Admin APIs by incorporating resource functions.
+- With the new changes the package name is changed to `docusign.dsadmin` which was previously maintained under the name `docusign.admin`.
+- Add the documentation and examples.
```
Comment (nipunayf) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:288]: Is this relevant to the release note? There are a few occurrences of this.

Comment (NipunaRanasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:288]: I also don't think we need to specifically mention about the doc improvements, which is common for all the revamped connectors(in that case we'll end up repeating the same point under every connector).

Comment (ThisaruGuruge) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md]: We can mention in the top section that the revamping effort includes documentation and examples improvements including making the documentation and examples consistent across the connectors.

Comment (NipunaRanasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md]: > We can mention in the top section that the revamping effort includes documentation and examples improvements including making the documentation and examples consistent across the connectors.

yeah that makes more sense. +1 for the suggestion.

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -262,22 +262,49 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
...
+
+#### `github` package
 
 - Introduced support for connecting to GitHub REST API version 2022-11-28, replacing the GitHub GraphQL API based connector.
 
-#### `guidewire.insnow` package 
+#### `googleapis.gcalendar` package
+
+- Enhanced the connector APIs by incorporating resource functions.
+- The package name has been changed to `googleapis.gcalendar` from its previous name, `googleapis.calendar`.
```
Comment (ThisaruGuruge) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:305]: HEre, we have to mention that the previous package is also available.

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -262,22 +262,50 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ### Revamped connector updates
 
-The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative. 
+The following new connectors along with major updates have been added to the Ballerina library, as part of the Ballerina connector revamp initiative.
 All the connectors listed below have been released under new major versions, due to significant changes in the APIs and functionalities.
+These updates also include improvements to the documentation and examples.
```
Comment (NipunaRanasinghe) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:267]:
```suggestion
All listed connectors have been released under new major versions, featuring significant API and functionality changes, along with improved documentation and examples.
```

**Review by ThisaruGuruge — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]
